TaylorFramework/Modules/caprices/ProccessingMessage/Options/VerbosityOption.swift

```diff
@@ -35,20 +35,20 @@ struct VerbosityOption: InformationalOption {
     func verbosityLevelFromOption() -> VerbosityLevel {
         switch optionArgument {
         case VerbosityLevelInfo:
-            return VerbosityLevel.Info
+            return VerbosityLevel.info
         case VerbosityLevelWarning:
-            return VerbosityLevel.Warning
+            return VerbosityLevel.warning
         default:
-            return VerbosityLevel.Error
+            return VerbosityLevel.error
         }
     }
     
-    func validateArgumentComponents(components: [String]) throws {
+    func validateArgumentComponents(_ components: [String]) throws {
         guard let firstElement = components.first else {
-            throw CommandLineError.InvalidInformationalOption("\nNo verbosity option specified")
+            throw CommandLineError.invalidInformationalOption("\nNo verbosity option specified")
```

Comment:
`enum CommandLineError: Error {` and it used in Caprice and Taylor modules, or i misunderstood you 😅?.